connect-widget/app/src/shared/api.ts

```diff
@@ -26,7 +26,11 @@ export function getApiToken(searchParams: URLSearchParams): string {
   const apiToken = searchParams.get(Constants.TOKEN_PARAM);
   if (!apiToken) {
     throw new Error(
-      `Missing query param ${Constants.TOKEN_PARAM}! To learn more go to the Connect Widget overview in our documentation.`
+      `Missing query param ${Constants.TOKEN_PARAM}! To learn more, go to the [Connect Widget](https://docs.metriport.com/devices-api/getting-started/connect-widget#token) overview in our documentation.`
+    );
+  } else if (apiToken === "demo") {
```

Comment:
We use `token === "demo"` in a lot of places in the widget (not the best practice) - would suggest creating a shared utility function that we can use in all places instead of hardcoding this check, and updating existing places accordingly. For example:

```
function isDemoToken(token: string): boolean {
    return token === "demo";
}
```